README.md

```diff
@@ -14,14 +14,30 @@ These instructions apply to minor and patch version updates. Major versions need
 a customized adaptation.
 
 After the CI is green:
-  * Update the version in the podspec
-  * Add the CocoaPods tag
-    * `git tag CocoaPods-{version}`
-    * `git push origin CocoaPods-{version}`
-  * Push the podspec to SpecsStaging
-    * `pod repo push staging GoogleDataTransport.podspec`
-  * Run Firebase CI by waiting until next nightly or adding a PR that touches `Gemfile`
-  * On google3, copybara and run a global TAP which should kick off automatically after each PR.
+* Update the version in the podspec to match the latest entry in the [CHANGELOG.md](CHANGELOG.md)
+* Checkout the `main` branch and ensure it is up to date.
+  ```console
+  git checkout main
+  git pull
+  ```
+* Add the CocoaPods tag (`{version}` will be the latest version in the [podspec](GoogleDataTransport.podspec#L3))
+  ```console
+  git tag CocoaPods-{version}
+  git push origin CocoaPods-{version}
+  ```
+* Push the podspec to SpecsStaging
```

Comment:
Might be worth writing that it should push to SpecsStaging if the GDT is intended to launch before or with the next Firebase release. Otherwise it should push to SpecsDev

Reply:
I added collapsible sections to hopefully make the reader's actions align with their intent and prevent accidental pod repo pushes to the wrong repo

Reply:
@paulb777  Should something similar be done on the GoogleUtils repo as well?

Reply:
Yes please.